next-cmake/CMakeLists.txt

```diff
@@ -198,7 +198,7 @@ if(HIPBLASLT_ENABLE_HOST)
 
     target_link_libraries(hipblaslt
         PUBLIC
-            ${hipblas_target}
+            roc::${hipblas_target}
```

Comment:
Nit for the future: when using non-optional string substitution like this, it can be a little better to quote it. That way you at least get an error about some library not being found at generation time vs missing usage requirements.